components/bounty/status-progress/controller.tsx

```diff
@@ -23,14 +21,9 @@ export default function BountyStatusProgress({ currentBounty }: { currentBounty:
     t("bounty:steps.closed"),
   ]);
 
-  const { service: daoService } = useDaoStore();
   const marketplace = useMarketplace();
 
-  const getChainTime = () =>
-    daoService
-      ?.getTimeChain()
-      .then(setChainTime)
-      .catch(console.log);
+  const chainTime = Date.parse(new Date().toISOString())
```

Comment:
`new Date().toISOString()` should be enough

Reply:
I received different returns
![image](https://github.com/layerx-labs/webnetwork/assets/34746557/663c54f9-6f23-4a53-95ab-05efb1ed61a6)